docs/AccountApi.md

```diff
@@ -0,0 +1,431 @@
+# ShipEngineSDK.Api.AccountApi
```

Comment:
Are these docs intendede to be published somewhere, or are they for browsing on github?

We should make it very clear they are auto-generated and not intended to be modified. Maybe using [front matter](https://jekyllrb.com/docs/front-matter/)?

Reply:
They're just meant for browsing on Github (or in your local repo).  Front matter looks like it'll work on GH but I'll see what it looks like rendering locally.  I also came across a type of Markdown comment that I'll try as well to see what works & looks better.